app/models/viewmodels/LiabilityViewModel.scala

```diff
@@ -16,29 +16,40 @@
 
 package models.viewmodels
 
+import models.CaseStatus.CaseStatus
 import models._
 
 case class LiabilityViewModel(
                                caseHeaderViewModel: CaseHeaderViewModel,
-                               isNewCase: Boolean,
                                hasPermissions: Boolean,
-                               showRulingAndKeywordsTabs: Boolean,
                                showChangeCaseStatus: Boolean,
                                showTakeOffReferral: Boolean,
-                               showReopen: Boolean
+                               showReopen: Boolean,
+                               caseStatus: CaseStatus
                              ) {
 
-  def showActionThisCase: Boolean = {
-    isNewCase && hasPermissions
-  }
+  def isNewCase: Boolean = caseStatus == CaseStatus.NEW
+
+  def showRulingAndKeywordsTabs: Boolean =
+    caseStatus == CaseStatus.OPEN ||
+      caseStatus == CaseStatus.REFERRED ||
+      caseStatus == CaseStatus.REJECTED ||
+      caseStatus == CaseStatus.SUSPENDED ||
+      caseStatus == CaseStatus.COMPLETED
```

Comment:
Maybe we could use `Set(CaseStatus.OPEN, CaseStatus.REFERRED, /* etc */).contains(caseStatus)` here?

Reply:
If you think it does make big difference I can make this change, to me there is no difference

Reply:
done :)

Reply:
Haha I'm not sure it makes a big difference either. :) I have a distant memory that we already have Sets of statuses somewhere in the code.